src/app/shared/entry.ts

```diff
@@ -282,6 +305,21 @@ export abstract class Entry implements OnInit, OnDestroy {
     }
   }
 
+  tagSorting(a: Tag, b: Tag): number {
+    const verifiedSorting = this.verifiedSorting(a, b);
+    if (verifiedSorting !== 0) {
+      return verifiedSorting;
+    } else {
+      if (a.last_built > b.last_built) {
```

Comment:
This is fine, but you could simplify the whole block to `return b.last_built - a.last_built`.